pkg/hintrunner/zero/zerohint_memcpy.go

```diff
@@ -3,9 +3,58 @@ package zero
 import (
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/core"
 	"github.com/NethermindEth/cairo-vm-go/pkg/hintrunner/hinter"
+	"github.com/NethermindEth/cairo-vm-go/pkg/utils"
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+
 	VM "github.com/NethermindEth/cairo-vm-go/pkg/vm"
 )
 
+func newMemcpyContinueCopyingHinter(n, dst hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "MemcpyContinueCopying",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//n -= 1
+			//ids.continue_copying = 1 if n > 0 else 0
+			var lhs fp.Element
```

Comment:
Do we need this variable? In the ```AssignVariable("n")``` we can just create new felt and set it to n-1, like this:
```ctx.ScopeManager.AssignVariable("n", new(fp.Element).Sub(n.(*fp.Element), &utils.FeltOne))

Reply:
Hey @MaksymMalicki for me the code block you added in your comment is empty

Reply:
@MaksymMalicki same for me, there is no code in that block 😢

Reply:
@danielcdz  i think you can simplify and do something like this : 

```
n, err := ctx.ScopeManager.GetVariableValue("n")
if err != nil {
	return err
}

ctx.ScopeManager.AssignVariable("n", n.(uint64)-1)
``` 

so that you dont need to declare any additionnal variable.

Reply:
and then : 

```
var v memory.MemoryValue
if n.(uint64) > 0 {
	v = memory.MemoryValueFromFieldElement(&utils.FeltZero)
} else {
	v = memory.MemoryValueFromFieldElement(&utils.FeltOne)
}
```